crates/astria-composer/src/executor/mod.rs

```diff
@@ -421,9 +424,7 @@ impl Future for SubmitFut {
                     .into_signed(this.signing_key);
                     debug!(
```

Comment:
Given the submitting things to sequencer is the main task of composer, I think submitting this as `info!` is justified.